config/environments/production.rb

```diff
@@ -54,7 +54,11 @@
   # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  ssl = %w[true yes 1].include?(ENV['RAILS_HOST_SSL'])
```

Comment:
I think you should use `ActiveModel::Type::Boolean.new.cast(ENV['RAILS_HOST_SSL'])` here as it supports even more common cases to represent a boolean with a sting.

Reply:
Yes, but then again it does cast 'no' to true